build.gradle.kts

```diff
@@ -52,7 +52,7 @@ tasks.register("apiKey") {
     doLast {
         val apiKeyFile = File("${System.getProperty("user.home")}/.gradle/gradle.properties")
         if (!apiKeyFile.exists()) {
-            print("Go to ${URL("https://developers.arcgis.com/dashboard/")} to get an API key.")
+            print("Go to the ${URL("https://links.esri.com/create-an-api-key")} to obtain a new API key access token. Ensure the following privileges are enabled: Basemaps, Geocoding, and Routing.")
```

Comment:
@shubham7109 I'm not familiar with Koltin... so I assume this reads OK in the print statement.  Thanks